bitnami/zookeeper/values-production.yaml

```diff
@@ -36,6 +36,14 @@ image:
 ##
 updateStrategy: RollingUpdate
 
+
+## Limits the number of pods of the replicated application that are down simultaneously from voluntary disruptions
```

Comment:
In values.yaml you also added the following sentence:

```
## The PDB will only be created if replicaCount is greater than 1
```

Can you add it also here?

Reply:
done